.github/workflows/build.yml

```diff
@@ -190,17 +191,10 @@ jobs:
   verify:
     name: Verify plugin
     needs: [ build ]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-8-cores
```

Comment:
Had to bump this as even with the GHA for maximizing build space it continued to run out of space running this job.